build/tasks.ts

```diff
@@ -260,4 +260,43 @@ const ensurePatched = register("vscode:patch", async (runner) => {
 	}
 });
 
+register("package", async (runner, releaseTag, binLocation) => {
+	if (!releaseTag) {
+		throw new Error("Please specify the release tag.");
+	}
+	if (!binLocation) {
+		throw new Error("Please specify the location of the binaries.");
+	}
+
+	const releasePath = path.resolve(__dirname, `../release/${releaseTag}`);
+	fse.removeSync(releasePath);
+	fse.mkdirpSync(releasePath);
+
+	await Promise.all([{
+		bin: "cli-linux",
+		name: "linux",
+	}, {
+		bin: "cli-osx",
+		name: "apple-darwin",
```

Comment:
Wouldn't we only want to do one of these on the specific platform?

Reply:
Ah, good point, currently this is set up so you download the two built binaries then package them up into archives. Would definitely be better to do this as part of the build process.